packages/moonstone/styles/variables.less

```diff
@@ -235,12 +235,13 @@
 @moon-formcheckbox-check-icon-size: 72px;
 @moon-formcheckbox-check-icon-size-large: 84px;
 @moon-formcheckbox-border-size: 2px;
-@moon-formcheckbox-border-radius: 3px;
+@moon-formcheckbox-border-radius: 960px; // Generic, sufficiently large value that converts to a nice amount of rem
 
 // Checkbox
 // ---------------------------------------
-@moon-checkbox-size: 30px;
-@moon-checkbox-size-large: 36px;
+@moon-checkbox-size: 27px;
```

Comment:
Should these size values update if we're keeping the round shape?

Reply:
I had considered that myself. It seemed (visually) fine when I looked at it after the switch back to round.

My interpretation of the GUI<->UX conversation response was that squares had visually implied something that they didn't want to possibly visually confuse, rather than the design not being able to be updated at all. I'm content with leaving the size as-is until we hear otherwise. What do you think?

Reply:
The only concern would be the parity with other apps (e.g. QML, web apps, etc.). Given that 3px width is a subtle difference, I'm okay to move forward with the change. I'll suggest to follow up with GUI to make sure we match the guideline, though.

Reply:
Good point. Will do followup.